main/GlobalInit.c

```diff
@@ -0,0 +1,20 @@
+
+#include"main.h"
+
+extern QueueHandle_t BufQueue1;
+extern SemaphoreHandle_t GetResponseSemaphore;
+
+
+
+
+
+void GlobalInit()
+{
+    ESP_ERROR_CHECK(nvs_flash_init());
+    ESP_ERROR_CHECK(esp_netif_init());
```

Comment:
should call from begining of its task or thread

Reply:
because I need them in all other component

Reply:
I fix it